app/lib/utils.ts

```diff
@@ -20,6 +22,10 @@ import type {
 import {PRODUCTS_AND_COLLECTIONS} from '~/queries/shopify/product';
 import type {I18nLocale} from '~/types/shopify';
 
+/** @see https://github.com/sanity-io/sanity/pull/4462 */
```

Comment:
This doesn't seem to the right `@see` URL?

Reply:
Haha no clue how I did that:
https://github.com/sanity-io/sanity/pull/4461